Sources/TripKitUI/view model/TKUINearbyViewModel.swift

```diff
@@ -114,7 +114,8 @@ public class TKUINearbyViewModel {
     
     self.availableModes = nearby
       .map { content -> [TKModeInfo] in
-        let allModes = content.locations.compactMap { $0.stopModeInfo }
+        var allModes = content.locations.compactMap { $0.stopModeInfo }
+        allModes = allModes.sorted(by: { (($0.identifier ?? "") < ($1.identifier ?? "")) })
```

Comment:
LGTM, minor style suggestion to use the `mutable` sorting variant as `allModes` is already a `var`:

```suggestion
        allModes.sort(by: { (($0.identifier ?? "") < ($1.identifier ?? "")) })
```